v3/js/iframe-embed.js

```diff
@@ -63,6 +75,7 @@ $(document).ready(function() {
   var heapPrimitivesBool = (queryStrOptions.heapPrimitives == 'true');
   var textRefsBool = (queryStrOptions.textReferences == 'true');
   var cumModeBool = (queryStrOptions.cumulative == 'true');
+  rawInputLst = queryStrOptions.rawInputLst;
```

Comment:
Please use local variables (var) or declare this as a global variable explicitly